wazimap_np/templates/homepage.html

```diff
@@ -77,25 +77,27 @@ <h2 class="column-full subhead">Exploring data</h2>
         <header class="column-full">
             <h1 class="article-header">About {{ WAZIMAP.name }}</h1>
 
-         <p class="explainer">
-            <p>NepalMap makes data on Nepal more accessible and understandable.</p>
-<p>NepalMap is of great use to journalists, students, academics and others who are interested in using data to learn about Nepal.</p>
-<p>This tool is a friendly and reliable resource for journalists who want to use data to tell stories or discover stories that are of public interest.</p>
-<p>NepalMap uses data from <a href="http://www.cbs.gov.np/" target="_blank">Central Bureau of Statistics</a>, and other official sources, to create user-friendly data visualizations on key demographic issues.</p>
-<p>You can </span><strong>embed, access or download</strong> the data too. You can also compare any two areas side-by-side. </span></p>
-<p>NepalMap is a project of <a href="http://codefornepal.org/en/"><strong>Code for Nepal</strong></a>, a non-profit that works to increase digital literacy and the use of open data in Nepal. Please make a </span><a href="https://www.paypal.com/fundraiser/charity/117735" target="_blank"><strong>tax deductible donation</strong></a> to help us maintain the server space for NepalMap, and empower Nepal digitally!</p>
-<p>Code for Nepal's founder, <a href="http://codefornepal.org/en/staff/ravinepal/" target="_blank">Ravi Kumar</a>, is grateful to all the volunteers who contributed to NepalMap! Code for Nepal is especially grateful to <a href="https://github.com/cliftonmcintosh" target="_blank">Clifton McIntosh</span></a>, for serving as the lead developer for this project. And to Code for Nepal&rsquo;s </span><a href="http://codefornepal.org/en/staff/amit-chaudhary/" target="_blank">Amit Chaudhary</a>.</p>
-</p>
+         <!-- <p class="explainer"> -->
+                <p>NepalMap makes data on Nepal more accessible and understandable.</p>
+                <p>NepalMap is of great use to journalists, students, academics and others who are interested in using data to learn about Nepal.</p>
+                <p>This tool is a friendly and reliable resource for journalists who want to use data to tell stories or discover stories that are of public interest.</p>
+                <p>NepalMap uses <a href="http://nationaldata.gov.np/" target="_blank">National Data Profile</a>, created by <a href="http://www.cbs.gov.np/" target="_blank">Central Bureau of Statistics</a>, and other official sources, to create user-friendly data visualizations on key demographic issues.</p>
```

Comment:
"the National Data Profile" instead of "National Data Profile"?
"the Central Bureau of Statistics" instead of "Central Bureau of Statistics"?

Having "the" before each one seems more idiomatic to me, but maybe that's a regional difference.

Reply:
Sure.